support-frontend/app/services/AuthenticationService.scala

```diff
@@ -1,38 +1,69 @@
 package services
 
-import com.gu.identity.cookie.IdentityKeys
-import com.gu.identity.play.AccessCredentials.{Cookies, Token}
-import com.gu.identity.play.AuthenticatedIdUser
-import com.gu.identity.play.AuthenticatedIdUser._
-import play.api.mvc.RequestHeader
+import cats.implicits._
+import com.gu.identity.auth.UserCredentials
+import com.gu.identity.model.User
+import com.gu.identity.play.IdentityPlayAuthService
+import config.Identity
+import org.http4s.Uri
+import play.api.mvc.{Cookie, RequestHeader}
 
 import scala.concurrent.{ExecutionContext, Future}
 
-class AuthenticationService(override val identityKeys: IdentityKeys) extends com.gu.identity.play.AuthenticationService {
-  override lazy val authenticatedIdUserProvider: Provider =
-    Cookies.authProvider(identityKeys).withDisplayNameProvider(Token.authProvider(identityKeys, "membership"))
+// The following classes were previously defined in identity-play-auth,
+// but have been removed as part of the changes to that library:
+// authenticating users via a call to identity API; removing periphery functionality.
+// They have been redefined here to reduce diff across PRs,
+// but these classes could get refactored / simplified / removed in subsequent PRs.
+sealed trait AccessCredentials
+object AccessCredentials {
+  case class Cookies(scGuU: String, guU: Option[String] = None) extends AccessCredentials {
+    val cookies: Seq[Cookie] = Seq(
+      Cookie(name = "SC_GU_U", scGuU)
+    ) ++ guU.map(c => Cookie(name = "GU_U", c))
+  }
+  case class Token(tokenText: String) extends AccessCredentials
 }
+case class IdMinimalUser(id: String, displayName: Option[String])
+case class AuthenticatedIdUser(credentials: AccessCredentials, user: IdMinimalUser)
 
 // TODO: consider porting this to identity-play-auth.
 class AsyncAuthenticationService(
-  authenticationService: AuthenticationService,
-  testUserService: TestUserService
+    identityPlayAuthService: IdentityPlayAuthService,
```

Comment:
This is the major change in this PR: the `IdentityPlayAuthService` now authenticates users by calling identity API.